runtime/src/lib.rs

```diff
@@ -41,13 +40,9 @@ use sp_std::prelude::*;
 #[cfg(feature = "std")]
 use sp_version::NativeVersion;
 use sp_version::RuntimeVersion;
-
+use pallet_xcm::XcmPassthrough;
```

Comment:
suggest move all these XCM imports after the last `pub use sp_runtime`
i.e. as they did here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L41, and use comment `// XCM Imports`

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231